modules/combined/test/tests/gap_heat_transfer_mortar/gap_heat_transfer_mortar_displaced.i

```diff
@@ -123,6 +119,28 @@
     boundary = 'right'
     value = 0
   [../]
+
+  [left_disp_x]
+    type = DirichletBC
+    preset = false
+    variable = disp_x
+    boundary = 'left'
+    value = .1
+  []
+  [right_disp_x]
+    type = DirichletBC
+    preset = false
+    variable = disp_x
+    boundary = 'right'
+    value = 0
+  []
+  [bottom_disp_y]
+    type = DirichletBC
+    preset = false
```

Comment:
This was copied over from @tophmatthews 's test. I will say that for non-zero Dirichlet values for displacements I really like to use non-preset because if you use preset then you can actually blow a Dirichlet node past a non-Dirichlet node and create inverted elements